### PR TITLE
feat: dark themed styles, form submission callback for asynch POST request

### PR DESCRIPTION
**Pull request by FlynnOConnell**
Capture

General dark themed form styling

Main callback function after form submission
- prevent default submission behavior (page reload) 
- password match validation
- asynch POST req to temporary endpoint with serialized JSON body

### CODE REVIEW

File: frontend/src/views/home.rs [hunk 1]
```diff
@@ -32,21 +32,88 @@ pub fn home() -> Html {
...
+            let email = email_ref.cast::<HtmlInputElement>().unwrap().value();
+            let password = password_ref.cast::<HtmlInputElement>().unwrap().value();
+            let confirm_password = confirm_password_ref
+                .cast::<HtmlInputElement>()
+                .unwrap()
+                .value();
+
+
+            // Fix: 
+            // Check password against DB entry -----
```
Comment (LukeBrandon) [inline frontend/src/views/home.rs:63]: what u mean here

File: frontend/src/views/home.rs [hunk 2]
```diff
@@ -32,21 +32,88 @@ pub fn home() -> Html {
...
+                    })
+                    .body(JsValue::from(
+                        serde_json::to_string(&registration_form).unwrap(),
+                    ))
+                    .send()
+                    .await
+                    .unwrap();
+
+                log::info!("post_request {:?}", &post_request);
+            });
```
Comment (LukeBrandon) [inline frontend/src/views/home.rs:100]: Is this just an example? Should be talking to our rocket server right?

Comment (FlynnOConnell) [reply frontend/src/views/home.rs:100]: Correct, that implementation is is another branch and is much more involved

File: frontend/src/views/home.rs (hunk 1 above)
Comment (FlynnOConnell) [reply frontend/src/views/home.rs:63]: It was more of a note-for-later that didn't need to be there

**Review by LukeBrandon — APPROVED** [review]
Surely we can find a better pattern for making web requests.  Getting something working is fine for now.

**Review by FlynnOConnell — COMMENTED** [review]
Dark theme good to ship

**Pull request merged**